js_modules/dagit/packages/core/src/workspace/VirtualizedScheduleTable.tsx

```diff
@@ -0,0 +1,336 @@
+import {gql, useLazyQuery} from '@apollo/client';
+import {Box, Button, Caption, Colors, Icon, Menu, Popover, Tag, Tooltip} from '@dagster-io/ui';
+import {useVirtualizer} from '@tanstack/react-virtual';
```

Comment:
this looks cool.  is the plan to move everything off of `react-virtualized`?

Reply:
I think that may make sense with `react-virtualized` now looking like it's in maintenance mode.